### PR TITLE
Add docker-compose.yml for local deployment

### PR DESCRIPTION
**Pull request by CermakM**
- use .env file to set up environment variables and credentials
- add .env file to .gitignore to prevent commiting private credentials

- modify README.md with instructions for local deployment

Signed-off-by: Marek Cermak <macermak@redhat.com>

new file:   docker-compose.yml
modified:   .gitignore
modified:   README.md

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    environment:
+      AWS_S3_ACCESS_KEY_ID: "${AWS_S3_ACCESS_KEY_ID}"
+      AWS_S3_SECRET_ACCESS_KEY: "${AWS_S3_SECRET_ACCESS_KEY}"
+      KRONOS_SCORING_REGION: "${KRONOS_SCORING_REGION}"  # [ECOSYSTEM] - <maven/pypi/npm>
+      DEPLOYMENT_PREFIX: "${DEPLOYMENT_PREFIX}"  # <DEV/STAGE/PROD> - affects where the data and log for training gets stored.
+      SERVICE_PORT: "6006"
+      SERVICE_TIMEOUT: "900"
+      SPARK_HOME: "/usr/local/spark"
+      PY4J_VERSION: "py4j-0.10.4-src.zip"
+      GREMLIN_REST_URL: "http://bayesian-gremlin-http-preview-b6ff-bayesian-preview.b6ff.rh-idev.openshiftapps.com"  # <url>:<port>
```
Comment (sara-02) [inline docker-compose.yml]: @CermakM It does not seem like a good idea to point to the staging graph for local development. As every query will make calls to AWS DynamoDB. 
To bring up your local-graph instance you need to `clone the data-models repo`
`cd local-setup`, and then do `docker-compose up`.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      FLASK_LOGGING_LEVEL: "DEBUG"
```
Comment (sara-02) [inline docker-compose.yml]: @CermakM We can remove this well, we added this in our local development docker-compose to help us debug issues as the application logs were not showing up.
Else, we can comment this as ask users to un-comment it for debugging.

**Comment by sara-02** [comment]
@CermakM I just realized that in the README we need to ask the users to install Spark as well. Otherwise local training and deployment will not work. Please add note about Spark installation in the README :)

File: docker-compose.yml (hunk 1 above)
Comment (CermakM) [reply docker-compose.yml]: Good point, will do. Thank you.

**Comment by sara-02** [comment]
@CermakM Thanks. Merging it now.

**Pull request merged**